### PR TITLE
Testing  task

### PR DESCRIPTION
**Pull request by lubov15911**

### CODE REVIEW

File: src/components/MovieCard/MovieCard.test.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import MovieCard from './MovieCard';
+
+describe('MovieCard', () => {
+    it('should render correctly', () => {
+        const mockData = {
```
Comment (Yana-Kalandarova) [inline src/components/MovieCard/MovieCard.test.js:8]: if you will use one mockData more than once it'll be nice to put it to the constants or mock

File: src/components/MoviesListItem/MoviesListItem.test.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        />);
+    });
+
+    it('should render correctly', () => {
+        expect(component).toMatchSnapshot();
+    });
+
+    it('should raise click event', () => {
+        component.find('li.movie-preview').simulate('click');
+        expect(fakeFunction).toHaveBeenCalledWith(354912);
```
Comment (Yana-Kalandarova) [inline src/components/MoviesListItem/MoviesListItem.test.js:32]: In my point of view, it's better do not hardcode 354912 and use `item.id`  instead

File: src/components/ResultsList/ResultsList.test.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import ResultsList from './ResultsList';
+
+describe('ResultsList', () => {
+    const fakeFunction = () => {};
```
Comment (Yana-Kalandarova) [inline src/components/ResultsList/ResultsList.test.js:7]: why did you use `fakeFunction = () => {};` in the one place and `fakeFunction = jest.fn()` in the other?
If it possible I suggest to use one of them

File: src/components/ResultsList/ResultsList.test.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+    it('should render correctly films array', () => {
+        const films = [{
```
Comment (Yana-Kalandarova) [inline src/components/ResultsList/ResultsList.test.js:10]: constants mocks?

File: src/components/ResultsList/ResultsList.test.js (hunk 3 above)
Comment (lubov15911) [reply src/components/ResultsList/ResultsList.test.js:7]: jest.fn() is a spy function, in cases when I don't need to spy a function I create a simple one like ()=>{}

**Pull request merged**